package.json

```diff
@@ -85,7 +84,7 @@
     "ember-papaparse": "0.0.3",
     "ember-pikaday": "2.2.1",
     "ember-promise-helpers": "1.0.2",
-    "ember-resolver": "^3.0.0",
+    "ember-resolver": "^2.0.3",
```

Comment:
I seem to recall not being comfortable with the semver breaking change. The old version is what is in the Ember blueprint. But honestly I don't remember.